scripts/apps/authoring/authoring/directives/AuthoringDirective.ts

```diff
@@ -138,6 +138,35 @@ export function AuthoringDirective(
                 }
             }, true);
 
+            function checkShortcutButtonAvailability(personal = false) {
+                if (personal && appConfig?.features?.publishFromPersonal) {
+                    return $scope.item.state === 'in_progress' || $scope.dirty;
+                }
+                return $scope.item.task && $scope.item.task.desk && $scope.item.state !== 'draft' || $scope.dirty;
+            }
+
+            // ToDesk -- Send an Item to a desk
+
+            // closeAndContinue -- Create an update of an item and Close the item.
+
+            // publish -- publish an item
+
+            // publishAndContinue -- Publish an item and Create an update.
```

Comment:
looks good, maybe those comments could be where you define those variables like:
```ts
$scope.toDeskEnabled = false; // Send an item to desk
```
otherwise this might later show up in some diffs and you wouldn't see the relation